apps/src/dropletUtils.js

```diff
@@ -131,11 +131,14 @@ standardConfig.categories = {
  * @param {DropletConfig} dropletConfig
  * @returns {Array}
```

Comment:
Would be good to include a param tag for otherConfig here, and maybe a description of what's in it. I wonder if we can come up with a more descriptive name than otherConfig?